test/helpers/UsingWitnetTestHelper.sol

```diff
@@ -43,7 +43,7 @@ contract UsingWitnetTestHelper is UsingWitnet {
 
   function _witnetCheckRequestResolved(uint256 _id) external view returns (bool) {
   // If the result of the data request in Witnet is not the default, then it means that it has been reported as resolved.
-  return wrb.readDrTxHash(_id) != 0;
```

Comment:
Not sure if this change affects user implementations of contracts that are `UsingWitnet`, but I prefer the old syntax because it is more simple.

Reply:
It's actually same login as before. This is test helper contract inheriting from UsingWitnet.

Reply:
This solution is much cheaper in terms of gas cost. Besides, this is just a test helper implementation.